src/Builder/DatagridBuilder.php

```diff
@@ -126,8 +128,7 @@ public function addFilter(DatagridInterface $datagrid, $type, FieldDescriptionIn
 
         $fieldDescription->mergeOption('field_options', ['required' => false]);
 
-        // NEXT_MAJOR: Remove first check (when requirement of Symfony is >= 2.8)
```

Comment:
which check was removed?

Reply:
None, it cannot be removed, will add this comment back

Reply:
Fixed!